packages/client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import IconButton from '@material-ui/core/IconButton';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import PropTypes from 'prop-types';
+import Modal from '../Modal';
+import Wizard from '../Wizard';
+import Button from '../Button';
+import styled from 'styled-components';
+import { COLORS } from '../../constants';
+import { withStyles } from "@material-ui/styles";
+
+
+const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
+
+const TextThemes = {
+  fontSize: {
+    default: 18,
+    lg: 22,
+  },
+  fontWeight: {
+    default: 'normal',
+    lg: 'bold',
+  }
+}
+
+const LinkStyled = styled(Link)`
+  font-family: Muli, sans-serif;
+  font-color: ${COLORS.blue};
+  padding-right: 40px;
+  font-size: ${(props) => TextThemes.fontSize[props.ver]}px;
+  font-weight: ${(props) => TextThemes.fontWeight[props.ver]};
+  text-decoration: none;
+
+  &:link {
+    color: ${COLORS.blue};
+  }
+  &:visited {
+    color: ${COLORS.blue};
+  }
+  &:hover {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+  &:active {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+`
+
+export default function NavBar(props)  {
+
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  let menuOpen = false;
+
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    menuOpen = false;
+    setAnchorEl(null);
+  };
+
+  let userLinks;
+  if (false) {
+    userLinks = (
+      <>
+        <LinkStyled href='/login' ver='default'>Login</LinkStyled>
```

Comment:
we should use React Router's link component instead instead so we can take advantage of the history/location hooks etc. Would need to replace `href='/login'` with `to='/login'`